great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -2245,6 +2243,87 @@ def _add_expectation_suite(
         self.expectations_store.set(key, expectation_suite, **kwargs)
         return expectation_suite
 
+    def update_expectation_suite(
```

Comment:
Do we need to be reaching out to Luigi regarding whether these are `public_api`?

Reply:
yup but that's a later ticket in the epic!